tensorflow/contrib/tensorrt/convert/convert_nodes.cc

```diff
@@ -693,9 +693,10 @@ class Converter {
       // TODO(jie): tf protobuf seems to be omitting the :0 suffix
       string output_name = node_def.name();
       if (i != 0) output_name = StrCat(output_name, ":", i);
-      if (output.is_tensor()) {
-        output.tensor()->setName(output_name.c_str());
-      }
+      // We should not call output.tensor()->setName(), since the name may have
```

Comment:
What if the name is not already set?

Reply:
Then it doesn't matter, we don't need to set the names for intermediate tensors, only for inputs and outputs of the engine, which is already done in ConvertSubGraphToEngine